pom.xml

```diff
@@ -50,7 +50,7 @@
     <mockito.version>1.10.19</mockito.version>
     <powermock.version>1.7.3</powermock.version>
 
-    <hazelcast.version>3.11</hazelcast.version>
+    <hazelcast.version>4.0-SNAPSHOT</hazelcast.version>
```

Comment:
I think that change is not necessary because we will still be using `master` profile in Jenkins build?
We will just make that 4.0 and send back a PR to master when hazelcast 4.0 is released.

Reply:
Yes, jenkins job uses `master` profile but `pr-builder` does not use `master` profile so I have changed main hazelcast version to see tests are passed or not.